code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm

```diff
@@ -71,10 +71,20 @@
 	desc = "A big, thick snake."
 	tt_desc = "E Reptilia Pythonidae"
 
-	icon_state = "python"
-	icon_living = "python"
-	icon_dead = "python_dead"
-	icon = 'icons/mob/snake_vr.dmi'
+	icon_state = "snek"
+	icon_living = "snek"
+	icon_dead = "snek_dead"
+	icon_rest = "snek_rest"
+	icon = 'modular_chomp/icons/mob/animal_ch.dmi' //CHOMPEdit - Newer, cooler sprites??
+
+	vore_active = 1  //CHOMPEdit - adding vore, about time :p
+	vore_capacity = 1
+	vore_default_mode = DM_HOLD
+	vore_icons = SA_ICON_LIVING | SA_ICON_REST
+	vore_escape_chance = 25
+	swallowTime = 50
+	vore_bump_chance = 10
+	vore_bump_emote = "slithers around and tries to devour"
```

Comment:
```suggestion
        // CHOMPEdit Start - New sprites & Added vore
        /*
        icon_state = "python"
	icon_living = "python"
	icon_dead = "python_dead"
	icon = 'icons/mob/snake_vr.dmi'
	*/
	icon_state = "snek"
	icon_living = "snek"
	icon_dead = "snek_dead"
	icon_rest = "snek_rest"
	icon = 'modular_chomp/icons/mob/animal_ch.dmi'

	vore_active = 1
	vore_capacity = 1
	vore_default_mode = DM_HOLD
	vore_icons = SA_ICON_LIVING | SA_ICON_REST
	vore_escape_chance = 25
	swallowTime = 50
	vore_bump_chance = 10
	vore_bump_emote = "slithers around and tries to devour"
	// CHOMPEdit End
```

Reply:
Bigger sections of code changes that are not in modular_chomp have to be marked with CHOMPEdit Start/End sections.

Reply:
thank you!!